atef/__init__.py

```diff
@@ -1,7 +1,3 @@
-from ._version import get_versions
-
-__version__ = get_versions()["version"]
-del get_versions
-
+from .version import __version__  # noqa: F401
```

Comment:
Commenting here because image files can't be commented on:

1. I think we need to add `*.jpeg` to `MANIFEST.in` (or figure out the modern equivalent)
2. The SLAC logo is close to a megabyte, which seems kinda excessive. Unfortunately it's already deep your git history, so changing it is probably not worth the effort at this point...

Reply:
Added to the manifest and replaced with a smaller file.  Removing the logo from the commit history is something we definitely can do, but would be a bit disruptive to our commit history